content/api/commands/screenshot.md

```diff
@@ -166,15 +166,17 @@ cy.screenshot('my-screenshot', {
 
 Screenshot naming follows these rules:
 
-- By default, a screenshot is saved to a file with a path relative to the
-  [screenshots folder](/guides/references/configuration#Folders-Files), appended
-  by a path relating to where the spec file exists, with a name including the
-  current test's suites and test name:
-  `{screenshotsFolder}/{specPath}/{testName}.png`
+- Screenshots are saved inside the
+  [screenshots folder](guides/core-concepts/writing-and-organizing-tests#Asset-File-Paths).
```

Comment:
Be sure to test your changes!

```suggestion
  [screenshots folder](/guides/core-concepts/writing-and-organizing-tests#Asset-File-Paths).
```

Reply:
Great catch. I just verified on the deploy preview the link is working as expected after https://github.com/cypress-io/cypress-documentation/pull/4326/commits/3065884613d79de02ff2b1d491554bd2459543d8